Section 508.md

```diff
@@ -0,0 +1,54 @@
+# Section 508
```

Comment:
This just describes the compliance to governmental standards w.r.t. accessibility for visually impaired users, which is added in this PR.